tools/opt/PassBuilder.cpp

```diff
@@ -0,0 +1,1172 @@
+//===- Parsing, selection, and construction of pass pipelines -------------===//
```

Comment:
There is already a comment in ` tools/opt/opt.cpp`. We need to use the PassBuilder defined in llvm_seahorn, otherwise `seaopt` will call the llvm pass rather than the seahorn passes. So I don't think we need this file.